### PR TITLE
NServiceBus and CloudEvents landing page

### PR DESCRIPTION
**Pull request by johnsimons**
This is part of https://github.com/Particular/CloudPlatformExperience/issues/318

The issue to collect feedback can be the existing https://github.com/Particular/NServiceBus/issues/5357 or create a new one?

I am considering leading the discussion with the following questions:

- Are you looking for better interoperability with native products like  [Azure EventGrid](https://learn.microsoft.com/en-us/azure/event-grid/cloudevents-schema), [AWS EventBridge via transformations](https://blog.direktiv.io/updated-create-cloudevents-from-aws-eventbridge-and-into-direktiv-8be003c31ae5) or [GCP EventArc](https://cloud.google.com/eventarc/docs/workflows/cloudevents)?
- Are you looking for interoperability with other messaging frameworks like MassTransit or Wolverine?
- What protocol bindings would you be using, for example, HTTP, AMQP, MATT, ANTS, or something else?
- Are you using other languages aside from dotnet csharp?
-

### CODE REVIEW

File: nservicebus/cloudevents.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: NServiceBus and CloudEvents
+summary: How to use NServiceBus with CloudEvents
+reviewed: 2024-09-16
+---
+
+[CloudEvents](https://cloudevents.io) is a [specification](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md) for describing event data in a common way. CloudEvents seeks to dramatically simplify event declaration and delivery across services, platforms, and beyond.
+
+NServiceBus currently does not natively support CloudEvents.
+
+&#87;e are currently capturing requirements in understanding customer needs when it comes to CloudEvents.
```
Comment (lailabougria) [inline nservicebus/cloudevents.md:11]:
```suggestion
We are currently capturing requirements to understand your needs regarding CloudEvents.
```

Comment (lailabougria) [reply nservicebus/cloudevents.md:11]: Since we're aiming to remove the personal voice check here anyway.

File: nservicebus/cloudevents.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+If you are interested in helping &#117;s shape how NServiceBus should support CloudEvents, drop &#117;s a comment in [this](https://github.com/Particular/NServiceBus/issues/5357) GitHub issue.
```
Comment (lailabougria) [inline nservicebus/cloudevents.md:12]:
```suggestion
If you are interested in helping us shape how NServiceBus should support CloudEvents by sharing some of your use cases, drop us a comment in [this](https://github.com/Particular/NServiceBus/issues/5357) GitHub issue.
```

**Review by lailabougria — COMMENTED** [review]
I assume this will be moved into the future folder pending the other PR.

**Comment by johnsimons** [comment]
@lailabougria, @mikeminutillo and I added an Intro page as well

**Comment by johnsimons** [comment]
Also @lailabougria we were considering adding images eg the CloudEvents logo, but we are unsure whether we can legally. Thoughts?

File: shape-the-future/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Shape the future with us
+summary: Provide feedback for the future of the Particular Service Platform
+reviewed: 2024-09-16
+---
+
+We are interested in your feedback regarding some technologies that we are currently looking at integrating into the Particular Service Platform.
```
Comment (lailabougria) [inline shape-the-future/index.md:7]:
```suggestion
We are interested in your feedback regarding some capabilities that we are considering integrating into the Particular Service Platform.
```

File: shape-the-future/index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Shape the future with us
```
Comment (lailabougria) [inline shape-the-future/index.md:2]:
```suggestion
title: Help us shape the future
```

File: shape-the-future/index.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+summary: Provide feedback for the future of the Particular Service Platform
+reviewed: 2024-09-16
+---
+
+We are interested in your feedback regarding some technologies that we are currently looking at integrating into the Particular Service Platform.
+
+This section contains some ideas that we have but if you have some other ideas that are not listed here please [tell us more](https://github.com/Particular/NServiceBus/issues/new?assignees=&labels=Feature&projects=&template=feature_request.yml).
```
Comment (lailabougria) [inline shape-the-future/index.md:9]:
```suggestion
This section contains some capabilities that we're currently considering, but if you have some other ideas that are not listed here, please [tell us more](https://github.com/Particular/NServiceBus/issues/new?assignees=&labels=Feature&projects=&template=feature_request.yml).
```

**Review by lailabougria — APPROVED** [review]

**Comment by lailabougria** [comment]
@johnsimons I noticed the questions listed in the PR weren't incorporated on the landing page, was that intentional?

**Comment by johnsimons** [comment]
@lailabougria we did see https://github.com/Particular/NServiceBus/issues/7159

**Comment by lailabougria** [comment]
Ah I see! 👍🏼

**Review by lailabougria — APPROVED** [review]

File: shape-the-future/index.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+title: Help us shape the future
+summary: Provide feedback for the future of the Particular Service Platform
+reviewed: 2024-09-16
+---
+
+We are interested in your feedback regarding some capabilities that we are considering integrating into the Particular Service Platform.
```
Comment (mikeminutillo) [inline shape-the-future/index.md:7]:
```suggestion
We are interested in your feedback regarding the future of the Particular Service Platform.
```

**Review by mikeminutillo — APPROVED** [review]

**Pull request merged**